apps/cpp_rtvm/main.cc

```diff
@@ -192,59 +226,171 @@ int ExecuteModel(ToolArgs& args) {
 #endif
 
   // Initialize TVM Runner
-  TVMRunner runner = TVMRunner(args.model, args.device);
+  auto runner = new TVMRunner(args.model, args.device);
```

Comment:
Why do you need this change? Probably I missed something, but I didn't see any place where pointer to `runner` is necessary. In case if you need the pointer to `runner`, I would recommend to use smart pointers instead of raw pointer.

Reply:
Handled. Don't need a pointer for now.